scripts/release/git_helpers.py

```diff
@@ -0,0 +1,28 @@
+from os.path import join
+from subprocess import run, PIPE
+
+import re
+
+
+def get_submodule_version(path):
+    full_path = join("submodules", path)
+    result = run(["git", "-C", full_path, "rev-parse", "--short=7", "HEAD"],
+                 stdout=PIPE, check=True, universal_newlines=True)
+    return result.stdout.strip()
+
+
+def get_past_submodule_version(path, master_repo_version):
```

Comment:
I suspect there's a simpler way of doing this - there has been every time so far! Would appreciate your input, @richfitz.

Reply:
Apparently this is the right way of doing it (you're trying to find the SHA for a submodule when the master was at `master_repo_version`, right?).  Odd that it's such an ugly command

Reply:
:+1: